torch/lib/c10d/FileStore.cpp

```diff
@@ -279,4 +307,13 @@ void FileStore::wait(
   }
 }
 
+void FileStore::checkKey(const std::string& key) {
+  if (!cleanupMode_ && key == cleanupKey_) {
+    throw std::runtime_error(
+        "Using FileStore's internal key: " + key +
+        " is "
+        "not supported, please use a different key");
+  }
+}
+
```

Comment:
Instead of having an internal key and doing a check for every operation, it would be cleaner to use a prefix for every op and then change the prefix in cleanup mode. Then this runtime error cannot happen and there is no need for a special key. E.g. initialize the default prefix as "/" and then change it to "cleanup/" in cleanup mode or something.